app/components/layers/useCapitalProjectsLayer.client.tsx

```diff
@@ -79,8 +97,15 @@ export function useCapitalProjectsLayer() {
       });
     },
     updateTriggers: {
```

Comment:
Because `DataFilterExtension` filters on on the GPU, not the CPU, it doesn't rely on the update loop that `updateTriggers` manages. I have this set to
```
updateTriggers: {
      getFillColor: [managingCode, capitalProjectId],
      getPointColor: [managingCode, capitalProjectId],
    },
```
and the filtering appears to be working with the rest of the changes outlined here.